### ISSUE TITLE
7.0.0 Integrate interval query functionality

### ISSUE THREAD

**Issue by stefanobranco** [elastic/elasticsearch-net#3680]
**Describe the feature**: Query DSL Support for new [intervals query](https://www.elastic.co/guide/en/elasticsearch/reference/7.0/query-dsl-intervals-query.html)

**Comment by jporzelt** [comment]
+1. This would be really cool!

**Comment by russcam** [comment]
This is scheduled to go into 6.7.0 as part of #3615 and will be ported to 7.x once `6.7` is merged into the `6.x` mainline

**Comment by russcam** [comment]
I made an error above; Intervals query was scheduled to go into 6.6.0 originally, but ended being merged into 7.0.0-beta1 on Elasticsearch side. This query will therefore being going into NEST and Elasticsearch.Net 7.x

**Comment by stefanobranco** [comment]
Alright, thank you for letting us know.

**Comment by russcam** [comment]
Closing this as #3711 is merged so this will be in the next release.
